scripts/build-json/build-page-json.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const bcd = require('./resolve-bcd');
+const examples = require('./compose-examples');
+const attributes = require('./compose-attributes');
+const prose = require('./slice-prose');
+const contributors = require('./resolve-contributors');
+
+const writeToFile = (propertyName, json) => {
+  const data = {
+    html: {
+      elements: {
+        [propertyName]: json,
+      }
+    }
```

Comment:
I'm not comfortable with these keys. 
First of all, I believe it's bad practice to put values as keys. I.e. `{"name": "Peter", "id": 12345}` is always better than `{"Peter": {"id": 12345}}`. 

More importantly, I don't understand the structure. 

It all starts with a URI (e.g. `/en-US/docs/Web/HTML/Element/video`) From that, I should use that slug, with some transform, to find the .json file. Perhaps I'd turn it into a file lookup for `./content/en-US/html/elements/video.json`

Does that make sense? Happy to debate.

Reply:
> I'm not comfortable with these keys.
First of all, I believe it's bad practice to put values as keys. I.e. {"name": "Peter", "id": 12345} is always better than {"Peter": {"id": 12345}}.

This intentionally mirrors the way browser-compat-data represents things, e.g. https://github.com/mdn/browser-compat-data/blob/master/html/elements/video.json . I think there's a lot of benefit to having a common way to address objects across repos.

> More importantly, I don't understand the structure.

This is a thing I don't understand yet about stumptown: the relationship between data object and directory structures in the repo on the one hand, and web pages and URL structures on the other.

I think they don't necessarily map directly on to each other (meaning, that we shouldn't assume that there's a simple generalised transformation from a URL to a stumptown path), and I think it might be a mistake to bake the assumption that they do directly map into the system.

MDN the website isn't envisaged to be the only user of stumptown. For example we'd like editors to be able to use the data to provide contextual help (as, for example VSCode already does with BCD and CSS data from the mdn/data repo). If the tie the repo structure to MDN URL structure, does that mean we have to break all the other users when we want to move MDN pages (which we do quite often)? 

> It all starts with a URI (e.g. /en-US/docs/Web/HTML/Element/video) From that, I should use that slug, with some transform, to find the .json file. Perhaps I'd turn it into a file lookup for ./content/en-US/html/elements/video.json

As I say I'm not sure about this, but I think I'd thought about this the other way around. That you start with the repo, and the meta.yaml files contain `mdn_url` or whatever, that tells you the URL for the page. Maybe that doesn't make sense though.